src/main/java/pl/kinga/task/TaskController.java

```diff
@@ -0,0 +1,77 @@
+package pl.kinga.task;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Controller
+@RequestMapping
+public class TaskController {
+
+    private TaskRepository taskRepository;
+    private List<Task> tasks;
```

Comment:
nie chcemy przechowywać listy tasków tutaj - to jest bardzo niebezpieczne takie pole;
musisz pamiętać, że te requesty get/post mogą zostać robione równolegle, i parę metod będzie się dostawało w tym samym czasie do Twojej listy, dlatego w takich przypadkach potrzebna jest synchronizacja. U Ciebie w zadaniu po prostu tego pola tu nie powinno być :)